src/amo/components/AddonsCard/index.js

```diff
@@ -44,6 +45,7 @@ type Props = {|
   footerLink?: Object | string | null,
   footerText?: string,
   header?: React.Node,
+  isHomepageCard?: boolean,
```

Comment:
I have updated this to be kept in `DefaultProps` and removed it from `Props`. I'm trying to remember if I was using `showPromotedBadge?`as an example - it was placed in both `DefaultProps` and `Props` as well (line 27 and line 40). Should I remove `showPromotedBadge?` in `Props`?

Reply:
I think `isHomepageCard` could be just in `Props` and not in `DefaultProps`, because it will default to `false` if omitted. I think the reason `showPromotedBadge` is in both places is because it is defaulted to `true` when `defaultProps` are set.

Reply:
Actually, I take all of that back! They can both stay in `DefaultProps` and be removed from `Props`.